### PR TITLE
[doc] document `enable_use_cgroup_memory_info`

### PR DESCRIPTION
**Pull request by johnpyp**
Option introduced in this PR: https://github.com/apache/doris/pull/35412

These are English only docs, as I do not speak Chinese. Not sure what the process is there, but feel free to push commits with the Chinese docs.

### CODE REVIEW

**Comment by KassieZ** [comment]
Hi, thank you for contributing to docs. It is recommended use:::tip Tips
This feature / parameter / configuration is supported since the Apache Doris XXX version
:::instead of use version tag (<version since="2.1.4"></version>),

**Comment by KassieZ** [comment]
No worries about CN docs, we will translate and update the docs as well.

File: versioned_docs/version-2.1/admin-manual/config/be-config.md [hunk 1]
```diff
@@ -1447,6 +1447,14 @@ Indicates how many tablets failed to load in the data directory. At the same tim
 * Description: Cgroups assigned to doris
 * Default value: empty
 
+#### `enable_use_cgroup_memory_info`
+
+<version since="2.1.4"></version>
```
Comment (morningman) [inline versioned_docs/version-2.1/admin-manual/config/be-config.md:1452]: Remove this kind of tag

**Comment by KassieZ** [comment]
Thank you for your contribution. However, this pull request contains comments and unresolved conflicts. As a result, it will be closed. If you still wish to submit your changes, please create a new pull request.

**Pull request closed without merging**